saleor/plugins/webhook/plugin.py

```diff
@@ -1479,6 +1479,7 @@ def get_taxes_for_checkout(
             parse_tax_data,
             checkout_info.checkout,
             self.requestor,
+            is_mutation=True,
```

Comment:
Are you sure that this is a correct way for handling this? Maybe I am missing something, but these hooks `get_taxes_for_checkout`, `get_taxes_for_order` are also called inside the query (in case when expiration time is exceeded). Maybe we could set this flag when we initialise the manager? Like `get_plugin_manager_promise` could determine if the request is query or mutation and pass this info directly to the `PluginsManager(is_mutation=<value>)`?

Reply:
I think it is the right way, just the name `is_mutation` is confusing. IMO this entire flag should be called `enforce_master_db` or something like this.